db/institutions.yml

```diff
@@ -241,4 +241,12 @@ sunybuffalo:
   name: The State University of New York at Buffalo
   state: ''
   zip: 14260
+nyu:
+  aka: [nyu, NYU, new york university, New York University]
+  city: New York
+  country: USA
+  departments: {chemistry: Department of Chemistry}
```

Comment:
departments is now a dict of dicts, but  I will pull this in and change it.  General structure (changed from before) is
departments: {
           chemistry: {name: Department of Chemistry, aka: [dept of funny smells, chem, poo dept]}
                       }